src/visuals/sprite.rs

```diff
@@ -164,6 +167,16 @@ impl Sprite {
         sprite
     }
 
+    /// Implement an `Animation` to a `Sprite`
+    ///
+    pub fn to_animation(&mut self, frame_count: u32, frame_delay: u32, loop_type: LoopType) {
```

Comment:
Do we really want a `to_animation` function? As discussed previously, once a Sprite, always a Sprite (and once an Animation, always an Animation)

Reply:
Yea, It's probably not useful at all, I don't see any use case. Will remove it